incubator/hnc/Makefile

```diff
@@ -289,14 +296,27 @@ exclude-system-namespaces:
 	@echo "If these tests fail due to the webhook not being ready, wait 30s and try again. Note that webhooks can take up to 30s to become ready"
 	@echo "after HNC is first deployed in a cluster."
 	@echo
+
+warn-hnc-repair:
+	@echo "************************************************************"
 ifndef HNC_REPAIR
-	@echo "******************************************************"
 	@echo "HNC_REPAIR IS NOT SET. CRITICAL TESTS WILL BE SKIPPED."
-	@echo "You have 5s to hit Ctrl-C to cancel and try again."
-	@echo "******************************************************"
 	@echo
-	@sleep 5
+	@echo "If HNC is installed via an operator (e.g. GKE Hierarchy"
+	@echo "Controller), this is probably what you want to do. Otherwise,"
+	@echo "consider setting HNC_REPAIR to run the full set of tests."
+else
+	@echo "HNC_REPAIR IS SET; ENSURE YOU HAVE PERMISSION TO DELETE HNC."
```

Comment:
Why `ensure you have permission to delete HNC` here?

Reply:
That's basically what HNC_REPAIR tests do - they delete components of HNC and repair them. Maybe "modify HNC configs" would be a better description here?

Reply:
Changed to "modify HNC configs."

Reply:
Ah I see, it's simply for running e2e test cases. I misunderstood with the scenario that HNC is installed by operator that you will clean it up by deleting HNC.

BTW just confirming, for the case of using operator with `HNC_REPAIR` not set, the reinstalling HNC will be trigger immediately by the operator when HNC is deleted and the missing `HNC_REPAIR` won't cause other e2e test failure, right?

Reply:
If HNC_REPAIR isn't set, we just skip all the tests that need us to break and/or reinstall HNC.